zap/src/output/tooling.rs

```diff
@@ -462,6 +462,11 @@ impl<'src> ToolingOutput<'src> {
 
 		self.push_unreliable_events();
 
+		self.push_line("else");
+		self.indent();
+		self.push_line("error(`Unknown remote instance: {remote_instance}`)");
+		self.dedent();
+
```

Comment:
We already check for remote instance at L391-394 of `tooling.rs`?

Reply:
Yeah, you're right. The error was only relevant because of the bug, now it's not.